ui/e2e/nightwatch.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Yeah, will do.. I was going to start adding as I go, I haven't really figured out the ideal "development workflow" for developing and iterating on tests with saucelabs yet.